packages/cli/src/commands/multisig/approve.ts

```diff
@@ -0,0 +1,47 @@
+import { flags } from '@oclif/command'
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+
+export default class ApproveMultiSig extends BaseCommand {
+  static description = 'Approves an existing transaction on a multi-sig contract'
+
+  static flags = {
+    ...BaseCommand.flags,
+    from: Flags.address({
+      required: true,
+    }),
+    for: Flags.address({
+      required: true,
```

Comment:
This would benefit from a description. Maybe something like this? 

```suggestion
      required: true,
      description: 'Address of the multi-sig contract',
```